staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/policy_decision.go

```diff
@@ -47,6 +47,28 @@ type PolicyDecision struct {
 	Elapsed    time.Duration
 }
 
+type PolicyAuditAnnotationAction string
+
+const (
+	// AuditAnnotationActionPublish indicates that the audit annotation should be
+	// published with the audit event.
+	AuditAnnotationActionPublish PolicyAuditAnnotationAction = "publish"
+	// AuditAnnotationActionError indicates that the valueExpression resulted
+	// in an error.
+	AuditAnnotationActionError PolicyAuditAnnotationAction = "error"
+	// AuditAnnotationActionExclude indicates that the audit annotation should be excluded
+	// because the valueExpression evaluated to null.
```

Comment:
Looks like it will be excluded not only because of being evaluated to null but also because of FailurePolicy set to Ignore. Better to be clear here.

Reply:
Sure, I've updated.